Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerQuoteBubbleView.swift

```diff
@@ -205,4 +214,16 @@ open class _ChatMessageComposerQuoteBubbleView<ExtraData: ExtraDataTypes>: _View
             }
         }
     }
+
+    private func setAttachmentPreview(hidden: Bool) {
+        if hidden {
+            layout.attachmentPreviewConstraints.prefix(3).forEach {
+                $0.constant = 0
+            }
+        } else {
+            layout.attachmentPreviewConstraints[0].constant = attachmentPreviewSize.width
+            layout.attachmentPreviewConstraints[1].constant = attachmentPreviewSize.height
+            layout.attachmentPreviewConstraints[2].constant = contentView.directionalLayoutMargins.leading
+        }
+    }
```

Comment:
I guess this is one of the cons of having an array of constraints instead of each constraint in the `Layout` struct? Maybe for this special case, we can store the exact constraints? Using the indices to hide or unhide this view is of course not a good idea because the array can change by the customer, or not do anything at all, if they are deactivated.

Reply:
Attempt here: https://github.com/GetStream/stream-chat-swift/pull/931/commits/1a5ac598099bfb40062b7bcd688bbdca2fafd648